src/prefect/server/schemas/schedules.py

```diff
@@ -365,7 +365,9 @@ class RRuleSchedule(PrefectBaseModel):
     model_config = ConfigDict(extra="forbid")
 
     rrule: str
-    timezone: Optional[TimeZone] = Field(default="UTC", examples=["America/New_York"])
+    timezone: Optional[TimeZone] = Field(
+        None, examples=["America/New_York"], validate_default=True
```

Comment:
I think we should keep the default `None` since the field is optional, and opt to use `validate_default`